content/projects/gsoc/index.adoc

```diff
@@ -22,42 +22,43 @@ In exchange, numerous Jenkins community members volunteer as "mentors" for the s
 
 image:/images/gsoc/opengraph.png[Jenkins GSoC, role=center, float=center]
 
-== GSoC 2025
+== GSoC 2026
 
-// We are planning to participate in Google Summer of Code in 2025. +
+ We are planning to participate in Google Summer of Code in 2026. 
 
-We are participating in Google Summer of Code in 2025. +
+// We are participating in Google Summer of Code in 2026. +
 // See our link:https://docs.google.com/document/d/1FYOBo12qz24Vxq0TxWuv9ElHH_rHP51ouMsPms4tTmw/edit?usp=sharing[Jenkins GSoC Mentoring Org Application Form].
-Google has accepted us as a mentoring organization in Google Summer of Code 2025.
-See our link:https://summerofcode.withgoogle.com/programs/2025/organizations/jenkins-wp[Jenkins organization page on the GSoC website].
+// Google has accepted us as a mentoring organization in Google Summer of Code 2026.
+// See our link:https://summerofcode.withgoogle.com/programs/2025/organizations/jenkins-wp[Jenkins organization page on the GSoC website].
 
 // Uncomment when application is worked on and submitted (Feb 2025)
 //(link:./2025/application[Jenkins GSoC Organisation Application Form])
 
-The five selected projects for 2025 are:
+The draft projects for 2026 are:
```

Comment:
```suggestion
// The draft projects for 2026 are:
```
As discussed, please comment out. I will take care of them over the next few days.